src/Backend/AMQPBackendDispatcher.php

```diff
@@ -295,4 +295,24 @@ protected function getApiQueueStatus()
 
         return json_decode($request->send()->getBody(true), true);
     }
+
+    protected function resolveDelayStrategy(DelayStrategyAware $factory): DelayStrategyAware
+    {
+        $delayStrategy = $this->settings['delay_strategy_class'];
+        if (
+            !empty($delayStrategy) &&
+            (
+                !class_exists($delayStrategy) ||
+                !(new \ReflectionClass($delayStrategy))->implementsInterface(DelayStrategy::class)
```

Comment:
Do you really need reflections for this?

Reply:
possible other checks are 

`in_array(DelayStrategy::class, class_implements($delayStrategy))` 

or 

`is_a($delayStrategy, DelayStrategy::class, true)` 

To be honest I have never used both of them, or even if I have I don't remember to have. Which one would you prefer? I can use this, and open another PR with some PHP7 syntax updates and update the other checks with reflection.

Reply:
I prefer the second one.